README.md

```diff
@@ -1,24 +1,225 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<div align="center">
+   <img src="./app/assets/images/rc-sd-logo.PNG" alt="Rudi Carrillo logo" width="23px" height="auto" />
+  <br/>
+    <h3><b> Actions App 📈</b></h3>
+     <img src="./app/assets/images/actions-logo-banner.PNG" alt="Actions App Banner" />
+  <br/>
 
-Things you may want to cover:
+</div>
 
-* Ruby version
+<!-- TABLE OF CONTENTS -->
 
-* System dependencies
+# 📗 Table of Contents
 
-* Configuration
+- [📖  Actions App](#--actions-)
+- [👁️ Deployment ](#-deployment-)
+- [👀 Sneak Peak (Author's App presentation)](#sneak-peak)
+- [🛠 Built With ](#-built-with-)
+   - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+- [💻 Getting Started ](#-getting-started-)
+   - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+- [👥 Authors ](#-authors-)
+- [🙏🏻  Acknowledgements ](#-acknowledgements-)
+- [🔮 Future Features](#-future-features)
+- [🤝 Contributing ](#-contributing-)
+- [⭐️ Show your support ](#️-show-your-support-)
+- [❓ FAQ (OPTIONAL) ](#-faq-optional-)
+- [📝 License ](#-license-)
 
-* Database creation
 
-* Database initialization
+<!-- PROJECT DESCRIPTION -->
+<div align="center">
 
-* How to run the test suite
+# 📖 Actions App
 
-* Services (job queues, cache servers, search engines, etc.)
+**"Elevate every exchange"** 
+</div>
+ <a name="about-project"></a>
 
-* Deployment instructions
 
-* ...
+mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
+
+<!-- Deployment -->
+
+## 👁️ Deployment <a name="deployment"></a>
+[Click here](https://actions-app.onrender.com) to see the Action's app deployment.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- Sneak Peak -->
+
```

Comment:
- [ ] Kindly deploy your app and provide the link in the README.

Reply:
@TedLivist hi, the sneak peak it has been on the readme file since I submitted the PR, not sure if you reviewed the wrong branch